app/processing/siti-agri/handle-known-defects/cap-bps-penalties/index.js

```diff
@@ -0,0 +1,16 @@
+const { recalculateBPSPenalties } = require('../cap-bps-penalties/recalculate-bps-penalties')
+const { bps } = require('../../../../schemes')
+const { P02, P04 } = require('../../../../constants/line-descriptions')
```

Comment:
I would probably define this constant at the top of the file - before the other two definitions